build.sh

```diff
@@ -18,27 +18,33 @@ PRINT_USAGE() {
     echo "build.sh [options]"
     echo ""
     echo "options:"
-    echo "--help, -h : Show help"
+    echo "--jobs, -j  : Multicore build (i.e. -j:3 for 3 cores)"
+    echo "--icu       : Path to ICU include folder (see example below)"
+    echo "--help, -h  : Show help"
     echo "--debug, -d : Debug build"
-    echo "CXX=<path to clang++>"
-    echo "CC=<path to clang>"
+    echo "--ccx=<path to clang++>"
+    echo "--cc=<path to clang>"
+    echo ""
+    echo "example : ./build.sh --ccx=/path/to/clang++ --cc:/path/to/clang -j=2"
```

Comment:
Could you please also change `:` to `=` in `--cc:...`?

`ccx` looks fine but deviates from `CXX` and harder to read the script. Maybe `cxx` is more consistent and better?

The printed options are now in descendent order. Should we reverse to ascendant?